api/helpers/db_helper.py

```diff
@@ -20,7 +20,7 @@ def get_class_by_tablename(tablename):
     :param tablename: String with name of table.
     :return: Class reference or None.
     """
-    for classObject in db.Model._decl_class_registry.values():
+    for classObject in db.Model.registry._class_registry.values():
```

Comment:
The _decl_class_registry was throwing error as in the latest version it is removed from the Model object. So updated it accordingly